sims/networks/basic/timing/GunnsRosesTiming.cpp

```diff
@@ -115,49 +121,59 @@ void GunnsRosesTiming::update(const double timestep)
                     N += 2;
                 }
             } else if (Gunns::GPU_DENSE == gpuMode) {
+                // Handle GPU DENSE mode
                 gpuTimePrev = gpuTime;
                 gpuTime = network->mSolveTimeAvg;
                 printf("    %9.2e", network->mSolveTimeAvg);
+                
                 if (errCheck) {
                     double error = 0.0;
-                    for (unsigned int i=0; i<5; ++i) {
-                        error += std::fabs(gpuPotentials[i] - cpuPotentials[i]);
+                    for (unsigned int i = 0; i < 5; ++i) {
+                        error += fabs(gpuPotentials[i] - cpuPotentials[i]);
                     }
                     printf("  %9.2e", error);
                 }
-                step = 0;
                 predictGpuThreshold();
                 gpuMode = Gunns::GPU_SPARSE;
             } else if (Gunns::GPU_SPARSE == gpuMode) {
+                // Handle GPU SPARSE mode
                 gpuSparseTimePrev = gpuSparseTime;
                 gpuSparseTime = network->mSolveTimeAvg;
                 printf("    %9.2e", network->mSolveTimeAvg);
+                
                 if (errCheck) {
                     double error = 0.0;
-                    for (unsigned int i=0; i<5; ++i) {
-                        error += std::fabs(gpuSparsePotentials[i] - cpuPotentials[i]);
```

Comment:
Please change to ```std::abs``` and resubmit.  We're trying to adhere to a new coding standard that explicitly uses the standard library namespace for all such math functions.

Reply:
Your latest commit didn't fix this, this is still showing fabs but needs to be std::fabs.  Please change to std::fabs and resubmit.